server/plugin.go

```diff
@@ -138,12 +139,40 @@ func (p *Plugin) GetURL() string {
 	return *config.ServiceSettings.SiteURL + "/plugins/" + pluginID
 }
 
+func (p *Plugin) OnDisconnectedTokenHandler(userID string) {
```

Comment:
Whenever a 401 is detected in the client, we get rid of the user token from the database and send him a message telling that he is disconnected and he should reconnect using the slash command.